README.md

```diff
@@ -1,12 +1,17 @@
-# cascade-agent
-  - create a package directory
-  - create a repo for the package
-  - `npm init` to set values for the package (the name of the package will be used when testing with `sudo npm link`)
-  - create the entry point js to put the app logic (this will be the contents of `tracing.js`)
-    - make sure to `module.export = <func>`
-
-# pre-requisites
-  - in order to view traces in aws x-ray, make sure to include the following aws credentials in app's  .env file
-    - `AWS_REGION`
-    - `AWS_ACCESS_KEY_ID`
-    - `AWS_SECRET_ACCESS_KEY`
+# Purpose
+This npm package is an auto-instrumentation agent for your applications to generate and export traces. It is a pre-requisite for Cascade users who wish to deploy their applications on AWS ECS with observability, and is compatible with AWS X-Ray as the tracing backend.
+
+# Set Up Tracing Agent
+1. install the cascade-agent package
+`npm install cascade-agent`
+
+2. require `cascade-agent` and specify the name of the service at the top of each service's server code
+`require('cascade-agent')('my-service');`
```

Comment:
Do you mean:

`require('cascade-agent', 'my-service')` ? 

Or maybe I'm mistaken?

Reply:
`require('cascade-agent')` returns a function that accepts a service name as an argument, so it's meant to be like that :)

Reply:
Ooh thanks!